server.ts

```diff
@@ -19,8 +19,149 @@ const {
 
 const agent = await login();
 
-function getEmbedImages(post: AtoprotoAPI.AppBskyFeedDefs.PostView) {
-  return AppBskyEmbedImages.isView(post.embed) ? post.embed.images : [];
+function getPost(
+  post: AtoprotoAPI.AppBskyFeedDefs.PostView,
+  fullMedia = false,
+) {
+  const author = post.author;
+
+  const text = (AppBskyFeedPost.isRecord(post.record))
+    ? processText(post.record)
+    : "";
+
+  const isReply = (AppBskyFeedPost.isRecord(post.record) && post.record.reply)
+    ? true
+    : false;
```

Comment:
This is clear, but redundant. I think true and false can be erased.
```suggestion
  const isReply = (AppBskyFeedPost.isRecord(post.record) && post.record.reply);
```